bin/tag-release-config.js

```diff
@@ -5,13 +5,17 @@ const utils = require("../src/utils");
 process.chdir(process.env.TR_DIRECTORY);
 
 const path = "./.tag-releaserc.json";
-return new Promise((resolve, reject) => {
-	editor(path, code => {
-		if (code === 0) {
-			const contents = utils.readFile(path);
-			resolve(contents);
-		} else {
-			reject(`Unable to edit ${path}`);
-		}
+const startTagRelease = () => {
```

Comment:
What need prompted you to wrap this in a function?

Reply:
It said that it didn't like only a returned and wanted a wrapped method. 🤔